ScoutSuite/core/server.py

```diff
@@ -0,0 +1,107 @@
+from sqlitedict import SqliteDict
+import cherrypy
+import cherrypy_cors
+
+import re
+
+count_re = re.compile(r".*_count$")
+
+class Server(object):
```

Comment:
This class lacks docstrings, and the method names are also very generic. It's a bit hard to understand what is going on. I think unit tests would be very appropriate and easy to implement.

Reply:
@zer0x64